src/app/interfaces/menu-item.interface.ts

```diff
@@ -14,6 +14,7 @@ export interface MenuItem {
   tooltip?: string;
   sub?: SubMenuItem[];
   isVisible$?: Observable<boolean>;
+  showTooltipAlways?: boolean;
```

Comment:
it's better to use `alwaysShowTooltip`, but it's not about this ticket, please check it once again.